src/server.h

```diff
@@ -674,6 +670,19 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 
 	// ModChannel manager
 	std::unique_ptr<ModChannelMgr> m_modchannel_mgr;
+
+	// Global server metrics backend
+	std::unique_ptr<MetricsBackend> m_metrics_backend;
+
+	// Server metrics
+	MetricCounterPtr m_uptime_counter;
```

Comment:
The old uptime counter was thread safe, is this?

Reply:
Prometheus is safe, current implementation no. i looked at the code, uptime is only called on ServerThread (passed as value to curl, or called from lua). Then the mutex is not needed

Reply:
but i agree in some context the thread safe implementation may be needed, i will put them as atomics in the non prometheus implementation